default/stringtables/en.txt

```diff
@@ -1402,6 +1402,15 @@ Count of minutes after the cookie record will be considered expired.
 OPTIONS_DB_PUBLISH_STATISTICS
 Enable sending empire staticstics to the player.
 
+OPTIONS_DB_FIRST_TURN_TIME
+Absolute time point in format "2019-01-20 11:59:59" in UTC. If empty, first turn advance will happen after expiring interval. Requires fixed interval to be enabled.
```

Comment:
Why does a fixed time for ending the first turn require fixed turn time intervals to be enabled...?

Reply:
If fixed interval disabled, `WaitingForTurnEnd` clears all timers set before. I couldn't quickly find solution to fix it.